git/02.development-workflow.md

```diff
@@ -18,6 +18,10 @@ supercede general best practices.
 Start by creating a branch from the [release branch](07.release-branching.md)
 and switching to it.
 
+```
+$ git checkout -b feature_branch_name
+```
```

Comment:
OK, just added a commit for this (and rebased the others onto `git-v2`). See what you think.